taskweaver/plugin/context.py

```diff
@@ -100,6 +100,10 @@ def log(self, level: LogErrorLevel, tag: str, message: str) -> None:
     def get_env(self, plugin_name: str, variable_name: str) -> str:
         """get an environment variable from the context"""
 
+    @abstractmethod
+    def wrap_text_with_delimiter_temporal(self, text: str) -> str:
+        """wrap text with delimiter for temporal data"""
+
```

Comment:
is it possible to also provide an implementation in the class below in the case of testing

Reply:
done